docs/provider/provider-load-balancing.md

```diff
@@ -0,0 +1,88 @@
+---
+slug: /provider-load-balancing
+title: Load Balancing
+---
+
+# Load Balancing
+
+## Overview
+
+Load balancing is a technique used to distribute incoming network traffic across multiple servers to ensure that no single server is overwhelmed with requests. This helps to improve the performance, reliability, and scalability of a service by spreading the load across multiple servers.
+
+If you wish to run multiple instances of the Lava RPC Provider service for the same chain, you can use a load balancer to distribute incoming requests across these instances. This can help to improve the performance and reliability of your service and increase your QoS.
+
+## Load Balancing in Lava
+
+When a client sends a request to the Lava RPC Provider service, it includes a custom header called `lava-lb-unique-id` in the request header.
+This header contains a unique identifier for the request, which is used by the load balancer to determine which backend server should handle the request.
+
+:::warning
+Ignoring this header when load balancing providers on the same chain with under the same wallet account will lead to consistency issues, as the load balancer will not be able to route requests from the same session to the same RPC Provider instance. This will affect your RPC Provider QoS score, and data reliability.
+:::
+
+## Recommended Setup
+
+We recommend using a load balancer that supports consistent hashing based on a unique request ID. This will ensure that requests from the same session are always routed to the same backend server, which is important for maintaining consistency in the service.
+
+You can use a variety of load balancers to achieve this, such as Nginx, Envoy, or any other load balancer that supports routing using the request headers and gRPC. In this guide, we will provide an example configuration for Nginx, which is a popular open-source web server and reverse proxy server.
```

Comment:
also add that this would be the official recommendation by Lava Foundation, using other proxies are possible but require deep understanding in the lava protocol and therefore include risk

Reply:
Done